modules/bookDesplay.js

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line no-unused-vars
+export default class Display {
+    books = [];
+
+    booksContainer = document.getElementById('containerBooks');
+
+    constructor() {
+      if (localStorage.getItem('books')) {
+        // eslint-disable-next-line no-undef
+        this.books = JSON.parse(localStorage.getItem('books')).map((book) => new Book(book.title, book.author, book.id));
+      }
+    }
+
+    addBook(newBook) {
+      this.books.push(newBook);
+      this.render();
+      this.saveBooks();
+    }
+
+    removeBook(id) {
+      this.books = this.books.filter((book) => book.id !== id);
+      this.render();
+      this.saveBooks();
+    }
```

Comment:
- [ ] I noticed that I can't add or remove a book. Whenever I try to, I get this error in the console. Kindly ensure that users can add or remove books in this milestone 👍 

![image](https://user-images.githubusercontent.com/77106962/222296214-098f2c37-7549-4d1e-9488-0123a8c2fa21.png)